src/js/tests/ffi.spec.ts

```diff
@@ -140,7 +162,7 @@ describe('bindings', () => {
     expect(() => {
       // This method will throw an error because scalars must be 32 bytes
       // but we gave it 1 byte
-      const commitment = context.commitToScalars([scalar])
```

Comment:
This is unrelated to this PR, linter complains about unused variable. This is okay because we are testing that calling the method will throw an error